lms/templates/navbar.html

```diff
@@ -38,12 +38,40 @@
             </li>
           </ul>
         </li>
-        <li class="nav-item">
-          <a href="/send" class="nav-link">
-            <i class="fa fa-upload" aria-hidden="true"></i>
-            {{ _('העלאת תרגילים') }}
+        <li class="nav-item dropdown">
+          <a id="courses-icon" href="#" class="nav-link" aria-expanded="false" aria-haspopup="true" aria-label="Courses" role="button" data-toggle="dropdown">
```

Comment:
Be sure to follow bootstrap 5.1 new syntax (I've already merged it, so you can take look at the currently existing code in main)